Tools/Scripts/webkitpy/style/checker.py

```diff
@@ -1251,7 +1251,10 @@ def process(self, lines, file_path, line_numbers=None):
 
         _log.debug("Using class: " + checker.__class__.__name__)
 
-        checker.check(lines)
+        current_error_count = self.error_count
+        output = checker.check(lines)
+        if isinstance(checker, SwiftChecker) and self.error_count > current_error_count:
+            _log.info("You can fix these errors with swift format --in-place \"%s\"" % file_path)
```

Comment:
Suggestion to better match phrasing of other output.
```suggestion
            _log.info("These errors can be fixed using swift format --in-place \"%s\"" % file_path)
```